### ISSUE TITLE
Userbuild commands still showing in context menu even if userbuild is disabled

### ISSUE THREAD

**Issue by oseam87** [IBM/zopeneditor-about#347]
## Development environment used

- Z Open Editor version: 3.1.1
- Editor Platform
  - [X ] Visual Studio Code
  - [ ] Red Hat CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standalone Theia
- Editor Platform Version: 1.78.2
- Operating System: Windows 10
- Java Version: 11
- [ ] Related to RSE API?
  - RSE API Plugin version:
  - Zowe CLI version:
  - Node.js version:
- Logs attached: no

## Problem Description

These items shouldn't be visible in the right-click context menu when userbuild is disabled:

"Run IBM User Build"
"Run IBM User Build with full upload"
"Run setup for IBM User Build"

There is probably a bug in the when clause of the menus.

**Comment by FALLAI-Denis** [comment]
Hi,

Missing parentheses?

```
{
  "command": "extension.userbuild",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm",
  "group": "zopeneditor@2"
},
{
  "command": "extension.setup",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm",
  "group": "zopeneditor@1"
},
{
  "command": "extension.full-userbuild",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm",
  "group": "zopeneditor@3"
}
```

Should be:

```
{
  "command": "extension.userbuild",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && (resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm)",
  "group": "zopeneditor@2"
},
{
  "command": "extension.setup",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && (resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm)",
  "group": "zopeneditor@1"
},
{
  "command": "extension.full-userbuild",
  "when": "!zopeneditor.isVSCodeWeb && config.zopeneditor.userbuild.enabled && (resourceLangId == cobol || resourceLangId == pl1 || resourceLangId == hlasm)",
  "group": "zopeneditor@3"
}
```

Same in `commandPalette`.

**Comment by phaumer** [comment]
Fixed in Z Open Editor 3.2.0.
